specs/protocol/revenue-sharing.md

```diff
@@ -0,0 +1,77 @@
+# Revenue Sharing
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+  - [Definitions](#definitions)
+  - [`RevenueSharer` predeploy](#revenuesharer-predeploy)
+    - [Invariants](#invariants)
+    - [Deploying `RevenueSharer`](#deploying-revenuesharer)
+      - [Existing Chains](#existing-chains)
+    - [Execution](#execution)
+  - [Simplified L1 Data Fee Expenditure](#simplified-l1-data-fee-expenditure)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Overview
+
+Revenue sharing is the process by which chains in the superchain contribute a portion of their revenue or profit to a beneficiary such as the Optimism collective. They may do this in return for support with the OP Stack and/or other benefits. 
+
+
+## Definitions
+| Term   | Name         | Definition  |
+| -------|--------------| ----------- |
+| $d$ | L1 Data Fee Revenue    | ETH transferred to the sequencer with L2 transactions to cover estimated L1 Data Fees(see below). Accumulates to `L1FeeVault`.
+| $e$ | L1 Data Fee Expenditure| ETH spent by the batcher on L1 to make transaction data available
+| $b$ | L2 Base Gas Revenue         | Base fee portion of L2 Gas Fee (ETH transferred to the sequencer to cover execution of L2 transactions). Accumulates to `BaseFeeVault`.
+| $q$ | L2 Priority Gas Revenue         | Priority fee portion of L2 Gas Fee (ETH transferred to the sequencer to cover execution of L2 transactions). Accumulates to `SequencerFeeVault`.
+| $g$ | L2 Gas Revenue         | $b + q$
+| $r$ | Sequencer Revenue      | $d + b + q$
+| $p$ | Sequencer Profit       | $r - e$
+| $s$ | Revenue share due to Optimism Collective | $\max(0.15r,0.025p)$
+
+## `RevenueSharer` predeploy
+Revenue sharing is achieved through an L2 [predeploy](./predeploys.md) contract `RevenueSharer` with address `0x4200000000000000000000000000000000000024`.
+
+### Invariants
+* The `RevenueSharer` contract should not accumulate any ETH or Tokens itself. 
```

Comment:
There should be a "Security Considerations" section at the bottom with responses to questions around the security, ie "what if a user sends ether or a token to the contract directly?" Will this mess with its accounting?

Reply:
@geoknee let me know if you want input on this section 👍🏻